tests/fake_messages/platformci_success_message.json

```diff
@@ -0,0 +1,55 @@
+    "priority": "4",
+    "message-id": "ID:messaging-devops-broker01.web.prod.ext.phx2.redhat.com-42692-1521074210015-2:573311:0:0:1",
+    "timestamp": "0",
+    "JMS_AMQP_ContentType": "text/json",
+    "JMS_AMQP_FirstAcquirer": "false",
+    "subscription": "/queue/Consumer.client-datanommer.openpaas-prod.VirtualTopic.eng.>"
+  },
+  "body": {
+    "msg": {
+      "category": "functional",
+      "status": "pass",
+      "ci": {
+        "url": "https://baseos-jenkins.rhev-ci-vms.datacenter.redhat.com",
+        "team": "BaseOS QE",
+        "irc": "#baseosci",
+        "email": "baseos-ci@redhat.com",
+        "name": "BaseOS CI"
+      },
+      "run": {
+        "debug": "https://baseos-jenkins.rhev-ci-vms.datacenter.redhat.com/job/ci-openstack/8465/artifact/citool-debug.txt",
+        "url": "https://baseos-jenkins.rhev-ci-vms.datacenter.redhat.com/job/ci-openstack/8465/",
+        "rebuild": "https://baseos-jenkins.rhev-ci-vms.datacenter.redhat.com/job/ci-openstack/8465/rebuild/parameterized",
+        "log": "https://baseos-jenkins.rhev-ci-vms.datacenter.redhat.com/job/ci-openstack/8465/console"
+      },
+      "recipients": [
+        "jscotka",
+        "ovasik"
+      ],
+      "system": [],
+      "artifact": {
+        "nvr": "setup-2.8.71-7.el7_4",
+        "scratch": true,
+        "component": "setup",
+        "source": "git://pkgs.devel.redhat.com/rpms/setup?#8d22fd57b66704a62e0db649dd372a1ce7bb06ee",
+        "branch": "rhel-7.4 rhel-7.3",
+        "type": "brew-build",
+        "id": "15667760",
+        "issuer": "ovasik"
+      },
+      "label": null,
+      "note": null,
+      "thread_id": "90d2bbe46220-1",
+      "xunit": "eJztWV1v4jgU/SsoL/PkuAmfsyqVgEbdbgtBSUbdeYpM4oKLsSPbQcP8+nU+SNMpUB462ma3L4nt3Ot7fM+1ORaXCkslU6JfV5dVu8XQBg8NofsCEaaMVv5paNjG1WUieIKFIplH2d6VDgskMZcRMTNzoHYJNlpbRNMXc8HjbtoopQpEFElZeS71U3FOzayRz0zYElbzAZEyhoXp7QcC3fDyiU6F4lssEKWgCFkFm498/5RbKqj5hNmaMAkWKaFx5blSKpF/QFiYgtLIFCu8BREB2400MVuaIk5tvc54hZQZ8Q184guoP+toTCoUreGg0+vCDAKsJzpbdqRnLvHACRfY9aHEKk2gjxhROzhGkkRA5h2QOWjaSGZ9LmtIRKtqPT8GvbDXOZWLmOiM88rhlOmKy+ccZx0wAHanC/rd/JMZYabJo9ryQI5OTbxFgiCmzgKxwGiNBdDES8LZWS4R17UVKbSgOHyxBuvCHJh6BWa/e7qgI5JgQOKzoikk13Vb++29cnbh6uJS6fOmmvBNQrHC8atKS5DQnkonqdYu3b6MR77j+uHkNgxG/p2v89Dt9fr93oXxBZ4yHxpKpPgNo3DiTufuzJkFQyMv7CP2zujO8e5vx6E3n4bX7sPs3h1dh1Mn+NO91pB26aYVE5159Va8bA364d04OqLeqBT0zQ74CSLEYhIjdQSw5wfeLAjv3Zub29nN0OgeNptx9kuhHDT7yx1r7NOx4/lnWHvO5HbuHHeAdfooXxbP1krgx3c+opCul0cUKag2idzdib935oUFi4KX0IJZMUtowwwEXCHBsJSm7hhlZdaH4MdAmXV4quoo60MfBOUTTwVD1PyxoXuU9aGsCArik5XGtn+Xln6+r1rF4VGeGh+kUApMeqRr2+221bYHxXrRNqozsu/++2wcA1yM66pJUvVIKK6jP/jtmTKYc/WSsiD/JW8EY1+bxtjX38LYhGLEmrLN2hcNI20P+B1Ig9URqVGhZa3VYltRqhBg65/YvqW1Aab9sGMynutkeI6tFJEpkk0hsqoYcK/mX+v6XFOTRaq0QoVyJxXeFCJ/r+X7n1q+kVre+tTyDdby/nc/cKah7wTf5vt0FSVIyeKwy4P7UIL6PndOJuHzorDO3k24KFiNuChY/+WLgvVaxdkfWsEcB/zOsrM6kEBDeGs3jbf2/+26cCAHnaaR1vl91wX4QsvD6v+Telte/QMHyMhu",
+      "type": "tier1"
+    }
+  }
+}
```

Comment:
Can you add a new line at the end of the file?

Reply:
Should be done now.